tests/common/dualtor/dual_tor_utils.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import pytest
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture(scope='session')
+def tor_mux_intf(duthosts):
+    '''
+    Returns the server-facing interface on the ToR to be used for testing 
+    '''
+    dut = duthosts[0]
+    config_facts = dut.config_facts(host=dut.hostname, source='running')['ansible_facts']
+    portchannel_members = get_portchannel_members(config_facts)
+    intfs = list(sorted(config_facts['PORT'].keys(), key=lambda intf: int(intf.replace('Ethernet', ''))))
+    for intf in intfs:
+        if intf not in portchannel_members:
+            logger.info("Using ToR interface {} for test".format(intf))
+            return intf
```

Comment:
Like the below comment for `get_portchannel_members`, this part can also be moved to `tests.common.devices` as get_vlan_ints or any other name, and then sort, return the first int?

Reply:
Additionally, is it always ok to return the first interface sorted by port sequence? Can this logic return Ethernet0, even when Ethernet0 is configured to be admin `down`. Do you want first admin UP interface instead?

Reply:
Good catch about admin `down`, will add a check for that and move.

Reply:
Thanks. I didn't see this check in the new changes. Is that not needed?

Reply:
I changed to take the test interface from VLAN members, which should always be up and connected to a server.